sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1971,7 +1971,6 @@ private void SetNameFieldsIfNull()
                         sourceIfUnmodifiedSince: sourceConditions?.IfUnmodifiedSince,
                         sourceIfMatch: sourceConditions?.IfMatch,
                         sourceIfNoneMatch: sourceConditions?.IfNoneMatch,
-                        sourceIfTags: sourceConditions?.TagConditions,
```

Comment:
this doesn't seem related.

Reply:
It’s not, that is to pass CI. I think we need to revert the swagger change that caused this.

Reply:
Resolved this issue.